src/Components/Search/Search.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './Search.css';
+
+class Search extends Component {
+  constructor(){
+    super()
+    this.state = {
+      city: ''
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({ city: event.target.value })
+  } 
+
+  render() {
+    return (
+      <div className="form-container">
+        <form className="form">
+          <input type="text" className="city-inp" placeholder="City" onChange={(event) => {
```

Comment:
I think we should change the className at a later point to city-input just so it is clear when we want to pull in this element.